hstream-kafka/hstream-kafka.cabal

```diff
@@ -148,14 +148,60 @@ library
     , stm
     , text
     , time
+    , unordered-containers
     , vector
     , yaml
     , Z-Data
-    , unordered-containers
 
   default-language:   GHC2021
   default-extensions:
     DerivingStrategies
     LambdaCase
     OverloadedStrings
     RecordWildCards
+
+executable hstream-kafka-cli
+  ghc-options:        -Wall -Wextra
+  main-is:            Main.hs
+  build-depends:
+    , aeson
+    , base                                                                 >=4.11 && <5
+    , brick
+    , bytestring
+    , haskeline
+    , hstream-common-base
+    , hstream-kafka:{hstream-kafka, hstream-kafka-client, kafka-protocol}
+    , network
+    , optparse-applicative
+    , text
+    , vector
+
+  hs-source-dirs:     app
+  default-language:   GHC2021
+  default-extensions: OverloadedStrings
+
+library hstream-kafka-client
```

Comment:
Consider merging your hstream-kafka-client library into hstream-kafka since there is no hstream-kafka-server library, we can let them all into one hstream-kafka.